src/stylist/__main__.py

```diff
@@ -126,5 +124,10 @@ def main(arguments):
         sys.exit(1)
 
 
+def main():
+    '''Main entry point.'''
+    return process(parse_cli())
+
+
 if __name__ == '__main__':
-    main(cli())
+    main()
```

Comment:
This change seems unnecessary. Can you explain the reason for it?

Reply:
This is to allow the `entry_points` setting in `setup.py` to work. Now that I look at it again, I can probably remove line 130-133 as well.

Reply:
Done as well.